### DISCUSSION TITLE
Question about scraping links from a archive.org search query with dynamic pages

### DISCUSSION THREAD

**Discussion by junguler** [projectdiscovery/katana#172] [Q&A] [answered]
hi, i've recently discovered this program and it seems to be exactly what i wanted, my goal is to scrape all the links from a certain search query from archive.org, the pages are dynamic and only seem to show 310 entries without needing to manually scroll down the page, here is an example page

```https://archive.org/details/librivoxaudio?and[]=year%3A%222022%22``` 

i just need the links to the pages themselves so depth 1 `-d 1` is enough, thank you

**Comment by ehsandeep** [comment, marked as answer]
thanks for sharing your input, and it's something that can be done or investigated after supporting - https://github.com/projectdiscovery/katana/pull/174

**Reply by junguler** [reply]
that's good to know, i'll look forward to it and see if/when it might be added, thanks for answering
